lib/av_client.ts

```diff
@@ -393,11 +394,24 @@ export class AVClient implements IAVClient {
           content: {}
       };
 
+      if (affidavit && this.electionConfig) {
+        try {
+          const encryptedAffidavit = JSON.parse(dhEncrypt(this.electionConfig.castRequestItemAttachmentEncryptionKey, affidavit).toString())
+          console.log(encryptedAffidavit)
+          castRequestItem.content['attachment'] = encryptedAffidavit.ciphertext
+          castRequestItem['attachment'] = `data:text/plain;base64,${encryptedAffidavit.ciphertext}`
```

Comment:
```suggestion
          const encryptedAffidavit = dhEncrypt(this.electionConfig.castRequestItemAttachmentEncryptionKey, affidavit).toString()
          console.log(encryptedAffidavit)
          castRequestItem.content['attachment'] = sha256(encryptedAffidavit)
          castRequestItem['attachment'] = `data:text/plain;base64,${encryptedAffidavit}`
```

`dhEncrypt#toString()` returns stringified (serialized) object, i suppose it should be easy to deserialize it on BE by using `Crypto::AES::DHPackage.from_string(encryptedAffidavit)`

@av-emil Could you confirm, please?

Reply:
This snippet is outdated. Do you still need my input, @w0st?

Reply:
Thanks, I'm fine now. The current solution works correctly with [this approach to decrypt (How to decrypt affidavits?
 @ Slack)](https://monterail.slack.com/archives/C0421MSKUKS/p1669216669333509?thread_ts=1669216475.509959&cid=C0421MSKUKS).

Reply:
@w0st, that slack link is on the monterail domain. I don't believe I have access there. 😅

Reply:
I wanted to show you this message ![image](https://user-images.githubusercontent.com/4086624/204272051-7b748d02-3f99-47de-85c3-5005926b43e5.png)